models/intermediate/trades/int_trade_agg_month.sql

```diff
@@ -120,7 +120,7 @@ with
     /* obtain aggregate function metrics for the asset pair */
     , trade_day_agg_group as (
         select
-            date('{{ dbt_airflow_macros.ts(timezone=none) }}') as day_agg
+            date('{{ dbt_airflow_macros.ts() }}') as day_agg
```

Comment:
Just wondering. Is there an issue with using timezones=none? I'd assume that's the same as the default UTC

Reply:
there isn't, but it is the default. My thought was why specify the default